### DISCUSSION TITLE
Add "System User" persona & Stories to Wiki

### DISCUSSION THREAD

**Discussion by waltage** [gcivil-nyu-org/team-2-inperson#51] [General]
Team, should we add these?  See the zenhub tab for the epics that have been tagged with persona "system"

**Comment by simaoalicechen** [comment]
I think it is a persona. If put it in Wiki, we will have to commit to the idea that the brainy part of this app is going to be about "ML". However, it seems that we have doubts over this. Maybe not add to this week's demo for now.

**Reply by waltage** [reply]
Beyond the algo tag, we have 5 epics for "system" tags 
https://github.com/gcivil-nyu-org/team-2-inperson/issues?q=is%3Aopen+label%3AEpic+label%3A%22PERSONA%3ASYSTEM%22+
